pogam/scrapers/seloger.py

```diff
@@ -161,6 +172,11 @@ def seloger(
     min_beds = floor(min_beds) if min_beds is not None else min_beds
     max_beds = ceil(max_beds) if max_beds is not None else max_beds
 
+    # convert code postal to insee code
+    insee_codes=[]
+    for n in post_codes:
```

Comment:
For very simple loops like these, list comprehensions are usually preferred, both for style and for performance reasons:

```python
insee_codes = [MapCPtoCI(cp) for cp in post_codes]
```
To be clear, for what we're doing here, what you have is perfectly fine.

Some references 
 -  https://leadsift.com/loop-map-list-comprehension/
 - https://towardsdatascience.com/how-list-comprehensions-can-help-your-code-look-better-and-run-smoother-3cf8f87172ae